pod-operation/src/components/inverter_board.rs

```diff
@@ -0,0 +1,18 @@
+use rppal::uart::{Parity, Uart};
+pub struct InverterBoard {
+	uart: Uart,
+}
+
+impl InverterBoard {
+	pub fn new() -> Self {
+		let uart = Uart::with_path("/dev/ttyACM0", 9600, Parity::None, 8, 1).unwrap();
+		Self { uart }
+	}
+
+	pub fn send_control(&mut self, velocity: f32, throttle: f32) {
```

Comment:
Chore: could we include a doc comment describing how this encodes and sends the message to the Pi Pico

Reply:
[Doc comments](https://doc.rust-lang.org/reference/comments.html#doc-comments) must begin with _three_ slashes.
Nitpick, compound adjective should be hyphenated (i.e. "space-separated")